docs/docs/getting-started.md

```diff
@@ -2,14 +2,18 @@
 
 Create a new project and start a local development environment:
 
-```sh
+```bash
 $ npx sku init my-app
 $ cd my-app
-$ npm start
+$ yarn start
 ```
 
-Don't have [npx](https://www.npmjs.com/package/npx)?
+By default, a new project's dependencies will be installed with the first supported package manager detected on your system.
+Package managers are detected in the following order: `yarn` -> `pnpm` -> `npm`.
```

Comment:
I wish we could detect what package is being used to run the CLI, but I haven't found a way to do that (yet).